scripts/Word-Cloud-Generator/Word_Cloud.py

```diff
@@ -0,0 +1,156 @@
+"""
+Install the following modules
+
+pip install wordcloud
+pip install fileupload
+pip install ipywidgets
+jupyter nbextension install --py --user fileupload
+jupyter nbextension enable --py fileupload
+
+"""
```

Comment:
also put this instruction in readme so end user can find out whats to install. Change this and your PR is ready to merge